### PR TITLE
Added note for multiple ids in one page

### PR DESCRIPTION
**Pull request by Ajayneethikannan**
Solving #3510

The change made in this PR is : 
Added a note for users that multiple ids cannot be present in a single page , and refer to classes instead.

@lmccart I have made the changes in the documentation as instructed, though it does not close the issue. I request you to review the changes ,

Thank you!

### CODE REVIEW

File: src/core/p5.Element.js [hunk 1]
```diff
@@ -122,7 +122,9 @@ p5.Element.prototype.parent = function(p) {
 /**
  *
  * Sets the ID of the element. If no ID argument is passed in, it instead
- * returns the current ID of the element.
+ * returns the current ID of the element.Note that only one element can have
```
Comment (limzykenneth) [inline src/core/p5.Element.js]: Missing space before "Note" here.

File: src/core/p5.Element.js [hunk 2]
```diff
@@ -122,7 +122,9 @@ p5.Element.prototype.parent = function(p) {
...
+ * a particular id in a page. Refer to <a href="#/p5.Element/class">.class()</a>
+ * function to set names for multiple elements.
```
Comment (limzykenneth) [inline src/core/p5.Element.js]: I personally wouldn't call it "name" as for input fields "name" is a very specific attribute and may cause confusion here.

Comment (Ajayneethikannan) [reply src/core/p5.Element.js]: @limzykenneth Thank you for the suggestion !
Would rephrasing the sentence as ,
".class() function can be used to identify multiple elements with the same class name." make it better ?

Comment (limzykenneth) [reply src/core/p5.Element.js]: Yeah that would be better

**Comment by Ajayneethikannan** [comment]
@limzykenneth I have done the suggested changes , thank you!

**Comment by lmccart** [comment]
thank you @Ajayneethikannan!

**Pull request merged**